lib/mirage.ml

```diff
@@ -471,10 +472,14 @@ module Entropy = struct
     append_main "module %s = %s" (module_name t) (construction t) ;
     newline_main () ;
     append_main "let %s () =" (name t);
-    append_main "  %s.connect %s" (module_name t) (id t);
+    append_main "  %s.connect %s >>= function" (module_name t) (id t);
+    append_main "  | `Error e -> %s" (driver_initialisation_error "entropy");
+    append_main "  | `Ok entropy ->";
+    append_main "  %s.handler entropy Nocrypto.Rng.Accumulator.add_rr >>= fun () ->" (module_name t);
```

Comment:
@pqwy is this the intended call semantics for `Nocrypto.Rng` feeding? shouldn't there be at least the named `source` parameter (as in [lwt](https://github.com/mirleft/ocaml-tls/blob/master/lwt/tls_lwt.ml#L277), [nocrypto.mli](https://github.com/mirleft/ocaml-nocrypto/blob/master/src/nocrypto.mli#L429), [rng.ml](https://github.com/mirleft/ocaml-nocrypto/blob/master/src/rng.ml#L139), [fortuna.ml](https://github.com/mirleft/ocaml-nocrypto/blob/master/src/fortuna.ml#L113))?
Could we please, before releasing `2.4.0` with TLS support, sit down and convince ourselves that the code is in a good shape (entropy is not completely broken, have a story for authenticators, ...)?